docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -242,8 +242,8 @@ Name                                                 Description
 ``pool.running_slots``                               Number of running slots in the pool. Metric with pool_name tagging.
 ``pool.deferred_slots.<pool_name>``                  Number of deferred slots in the pool
 ``pool.deferred_slots``                              Number of deferred slots in the pool. Metric with pool_name tagging.
-``pool.scheduled_tasks.<pool_name>``                 Number of scheduled tasks in the pool
-``pool.scheduled_tasks``                             Number of scheduled tasks in the pool. Metric with pool_name tagging.
+``pool.scheduled_slots.<pool_name>``                 Number of scheduled slots in the pool
+``pool.scheduled_slots``                             Number of scheduled slots in the pool. Metric with pool_name tagging.
```

Comment:
And this one from: https://github.com/apache/airflow/blob/b8a25b980265b26282f3414f1fdf5dab01fd5185/airflow/jobs/scheduler_job_runner.py#L1837